wordle-web/pages/PlayerScores.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-container fluid fill-height justify-center>
+    <v-card>
+      <v-card-title>
+        <h1 class="display-1">Player Stats</h1>
+      </v-card-title>
+      <v-card-text>
+        <v-simple-table>
+          <thead>
+            <tr>
+              <th>#</th>
+              <th>Name</th>
+              <th>Games</th>
+              <th>Average Attempts</th>
+              <th>Average Seconds</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="(stat, index) in stats" :key="index">
+              <td>{{ index + 1 }}</td>
+              <td>{{ stat.name }}</td>
+              <td>{{ stat.gameCount }}</td>
+              <td>{{ stat.averageAttempts }}</td>
+              <td>{{ stat.averageSeconds }}</td>
```

Comment:
If possible, it might be good to try and round this value (if its a decimal) to either 2 or 3 places to help make the page look cleaner. Otherwise everything looks good!